erts/doc/src/erlang.xml

```diff
@@ -1616,35 +1696,83 @@ b</pre>
 
     <func>
       <name name="error" arity="2" since=""/>
-      <fsummary>Stop execution with a specified reason.</fsummary>
+      <fsummary>Raise an exception of class <c>error</c></fsummary>
       <desc>
-        <p>Stops the execution of the calling process with the reason
-          <c><anno>Reason</anno></c>, where <c><anno>Reason</anno></c>
-          is any term. The exit reason is
+        <p>Raises an exception of class <c>error</c> with the reason
+        <c><anno>Reason</anno></c>, where <c><anno>Reason</anno></c>
+          is any term. The error reason is
           <c>{<anno>Reason</anno>, Where}</c>, where <c>Where</c>
           is a list of the functions most recently called (the current
           function first). <c><anno>Args</anno></c> is expected to be the
```

Comment:
The word "stacktrace" could be used here, since it is used in the documentation of raise/3 and probably other places.